src/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,602 @@
+// Navigation ------------- //
+
+$height-nav-secondary: 5rem;
+$width-nav-sidebar:    250px;
+
+
+.usa-header {
+  width: 100%;
+  z-index: 1;
```

Comment:
Are we setting z-indexes in variables yet? How are we keeping track that 1 makes sense here?